users/views.py

```diff
@@ -137,3 +140,70 @@ def post(self, request):
             return JsonResponse({'MESSAGE': 'KEY_ERROR'}, status=400)
         except TypeError:
             return JsonResponse({'MESSAGE': 'TYPE_ERROR'}, status=400)
+
+class FindUsernameView(View):
+
+    def post(self, request):
+        try:
+            data  = json.loads(request.body)
+            name  = data['name']
+            email = data['email']
+            
+            if not utils.validate_email(email):
+                return JsonResponse({'MESSAGE': 'INVALID_EMAIL'}, status=400)
+            
+            forgotten_user = User.objects.filter(name=name, email=email)
+            
+            if not forgotten_user.exists():
+                return JsonResponse({'MESSAGE': 'WRONG_USER'}, status=400)
+
+            forgotten_user = forgotten_user[0]
+            return JsonResponse({
+                "USERNAME"  : forgotten_user.username[:-3]+ "*" * 3,
+                "CREATED_AT": forgotten_user.created_at
+            }, status=200)
+        except json.decoder.JSONDecodeError:
+            return JsonResponse({'MESSAGE': 'JSON_DECODE_ERROR'}, status=400)
+        except KeyError:
+            return JsonResponse({'MESSAGE': 'KEY_ERROR'}, status=400)
+        except TypeError:
+            return JsonResponse({'MESSAGE': 'TYPE_ERROR'}, status=400)
+ 
+class MakeTemporaryPasswordView(View):
+
+    def post(self, request):
+        try:
+            data     = json.loads(request.body)
+            username = data['username']
+            name     = data['name']
+            email    = data['email']
+
+            if not utils.validate_username(username):
+                return JsonResponse({'MESSAGE': 'INVALID_USERNAME'}, status=400)
+
+            if not utils.validate_email(email):
+                return JsonResponse({'MESSAGE': 'INVALID_EMAIL'}, status=400)
+
+            forgotten_user = User.objects.filter(
+                username = username, 
+                name     = name, 
+                email    = email
+            )
```

Comment:
`Model의 objects manager class`의 `filter method`는 QuerySet을 반환합니다. 그렇기 때문에 변수명을 복수명사로 수정해주시면 좋을 것 같습니다.
```suggestion
            users = User.objects.filter(
                username = username, 
                name     = name, 
                email    = email
            )
```